### PR TITLE
tidy up archive modal

### PR DESCRIPTION
**Pull request by earl-warren**
- Make it consistent with the other modals of the dangerous actions.

Refs: https://codeberg.org/forgejo/forgejo/pulls/1432

(cherry picked from commit 8b1225f9742cc0d3942824895923cbc8e9d49d04)

### CODE REVIEW

File: templates/repo/settings/options.tmpl [hunk 1]
```diff
@@ -976,20 +976,23 @@
...
-				{{template "base/modal_actions_confirm" .}}
+				</div>
+				<form action="{{.Link}}" method="post">
+					{{.CsrfTokenHtml}}
+					<input type="hidden" name="action" value="{{if .Repository.IsArchived}}unarchive{{else}}archive{{end}}">
+					<input type="hidden" name="repo_id" value="{{.Repository.ID}}">
+					<div class="text right actions">
+						<button class="ui cancel button">{{.locale.Tr "settings.cancel"}}</button>
+						<button class="ui red button">{{.locale.Tr "repo.settings.archive.button"}}</button>
+					</div>
```
Comment (silverwind) [inline templates/repo/settings/options.tmpl:993]: Why is `modal_actions_confirm.tmpl` unsuitable?

File: templates/repo/settings/options.tmpl [hunk 2]
```diff
@@ -976,20 +976,23 @@
...
-			<form action="{{.Link}}" method="post">
-				{{.CsrfTokenHtml}}
-				<input type="hidden" name="action" value="{{if .Repository.IsArchived}}unarchive{{else}}archive{{end}}">
-				<input type="hidden" name="repo_id" value="{{.Repository.ID}}">
-				{{template "base/modal_actions_confirm" .}}
+				</div>
+				<form action="{{.Link}}" method="post">
+					{{.CsrfTokenHtml}}
+					<input type="hidden" name="action" value="{{if .Repository.IsArchived}}unarchive{{else}}archive{{end}}">
+					<input type="hidden" name="repo_id" value="{{.Repository.ID}}">
```
Comment (delvh) [inline templates/repo/settings/options.tmpl:989]: What happened in these lines?
Whitespace seems to have changed, but I don't understand why.

File: templates/repo/settings/options.tmpl [hunk 3]
```diff
@@ -976,20 +976,23 @@
 				{{end}}
 			</div>
 			<div class="content">
-				<p>
+				<div class="ui warning message">
```
Comment (wxiaoguang) [inline templates/repo/settings/options.tmpl:979]: It makes the UI looks unnecessarily complex by using such `warning message`.

I never see a dialog with inner warning color/border for the main content.

I think the old UI looks good enough, the content message is rendered as normal text:

image

File: templates/repo/settings/options.tmpl (hunk 2 above)
Comment (wxiaoguang) [reply templates/repo/settings/options.tmpl:989]: They would like to fix a "misalignment" bug, but the bug doesn't exist in Gitea anymore.

**Pull request closed without merging**